rwatch/ui/animation/property_animation.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+/* property_animation.h
+ * declarations for property animations
+ * libRebbleOS
+ *
+ * Author: Carson Katri <me@carsonkatri.com>
+ */
+
+#include "FreeRTOS.h"
+
+#include "librebble.h"
+#include "point.h"
+#include "rect.h"
+#include "size.h"
+#include "animation.h"
+
+#include "appmanager.h"
+
+// Use to easily calculate changes
+#define LERP(a, b)  ((a) + ((b) - (a)) * distance_normalized / ANIMATION_NORMALIZED_MAX)
```

Comment:
(informational; I'll merge anyway)

Would be better not to expose this symbol in the global namespace (other people may want a `LERP` define of their own).  Better to have this only inside the `.c` file.